tests/polycube_from_docker_to_host.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#
+# This script copies all the binaries, headers and libraries from a running Polycube docker image to the host's filesystem.
```

Comment:
Michel, 
1) the line is too long;
2) it is not enough to say what it does; it would be better to say also *why* this is needed.
Thanks!